### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Patrycja-J**
## Before submitting checklist

- [ ] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [x] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,11 +85,11 @@ Add a snippet to any step of your ML pipeline once. Decide what and how you want
 
 Organize logs in a fully customizable nested structure. Display model metadata in user-defined dashboard templates.
 
-* <a href="https://app.neptune.ai/o/showcase/org/onboarding-project/runs/table?viewId=98f66b32-2279-4b73-8210-863021c440ac&product_tour_id=444083"><b>Nested metadata structure:</b></a> flexible API lets you customize the metadata logging structure however you want. Talk to a dictionary at the code level. See the folder structure in the app. Organize nested parameter configs or the results on k-fold validation splits the way they should be.
+* <a href="https://docs.neptune.ai/about/namespaces_and_fields/"><b>Nested metadata structure:</b></a> flexible API lets you customize the metadata logging structure however you want. Talk to a dictionary at the code level. See the folder structure in the app. Organize nested parameter configs or the results on k-fold validation splits the way they should be.
```
Comment (normandy7) [inline README.md:88]: > Talk to a dictionary at the code level.

What does this mean? I've been meaning to ask about it before, but must have forgotten.

**Review by normandy7 — COMMENTED** [review]
One question; looks good otherwise.

File: README.md (hunk 1 above)
Comment (normandy7) [reply README.md:88]: Yes, I cannot make sense of it at all, so I think it would be good to revisit it.

Comment (Patrycja-J) [reply README.md]: I don't want to block all the changes in this README because of this sentence, so I removed it.

**Review by normandy7 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <div align="center">
-    <img src="https://raw.githubusercontent.com/neptune-ai/neptune-client/assets/readme/github-banner.jpeg" width="1500" />
-    &nbsp;
+    <img src="https://neptune.ai/wp-content/uploads/2023/08/Github-cover.png" width="1500" />
```
Comment (Raalsky) [inline README.md:2]: Please upload all the images to `assets` branch. @domikoz probably knows the details of the process here.

**Review by Raalsky — APPROVED** [review]

**Review by normandy7 — APPROVED** [review]

**Pull request merged**